packages/web-api/src/methods.ts

```diff
@@ -1642,6 +1644,18 @@ export abstract class Methods extends EventEmitter<WebClientEvent> {
     update: bindApiCall<ChatUpdateArguments, ChatUpdateResponse>(this, 'chat.update'),
   };
 
+  public readonly entity = {
+    /**
+     * @description
+     * Provide information about the entity to be displayed in the flexpane.
+     * A link to the API documentation will be added when this API is publicly released.
```

Comment:
```suggestion
     * @description Provide information about the entity to be displayed in the flexpane.
     * @see {@link https://docs.slack.dev/reference/methods/entity.presentDetails}
```

📚 suggestion: Perhaps this is too optimistic but we might want to inline the description before merging?